monitoring/ceph-mixin/prometheus_alerts.libsonnet

```diff
@@ -736,5 +736,76 @@
         },
       ],
     },
+    {
+      name: 'rbdmirror',
+      rules: [
+        {
+          alert: 'CephRBDMirrorConnectionLost',
+          'for': '1m',
+          expr: 'mirror_connection_state != 1',
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.1.2' },
+          annotations: {
+            documentation: '#TODO',
+            summary: 'RBD Mirror connection is lost, but mirror daemon is up',
+            description: "RBD Mirror connection is lost, but mirror daemon is up",
+          },
+        },
+        {
+          alert: 'CephRBDMirrorImagesPerDaemonHigh',
+          'for': '1m',
+          expr: 'count(rbd_mirror_image_status {daemon=xx) >= 100| |high image lag time|possible resource constraints or high data transfers|(rbd_mirror_image_status\{image=X, pool=Y, last_sync_duration)} == 1) > rbd_mirror_image_schedule{image=X, pool=Y, interval}',
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.1.2' },
+          annotations: {
+            documentation: '#TODO',
+            summary: '#TODO',
+            description: "#TODO",
+          },
+        },
+        {
+          alert: 'CephRBDMirrorImagesNotInSync',
+          'for': '1m',
+          expr: 'count(rbd_mirror_image_status !=1) > 0',
+          labels: { severity: 'warning', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.1.2' },
+          annotations: {
+            documentation: '#TODO',
+            summary: '#TODO',
+            description: "#TODO",
+          },
+        },
+        {
+          alert: 'CephRBDMirrorImagesNotInSync',
+          'for': '1m',
+          expr: 'count(rbd_mirror_image_status != 1) > (count(rbd_mirror_image_status) *.1)',
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.1.2' },
+          annotations: {
+            documentation: '#TODO',
+            summary: '#TODO',
+            description: "#TODO",
+          },
+        },
+        {
+          alert: 'CephRBDMirrorImageTransferBandwidthHigh',
+          'for': '1m',
+          expr: 'sum(rate(rbd_mirror_image_transfer_bytes[5m]) > %(CephRBDMirrorImageTransferBandwidthThreshold)s' % $._config,
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.1.2' },
+          annotations: {
+            documentation: '#TODO',
+            summary: '#TODO',
+            description: "#TODO",
+          },
+        },
+        {
+          alert: 'CephRBDMirrorLocalDaemonInactive',
+          'for': '1m',
+          expr: 'up{job="mirror_daemon_scrape_job"} == 0',
```

Comment:
I do not know if the scrape job for rbd mirror daemons has this name. We need to verify.

Reply:
Not sure about this `scrape job` but can we check the actual Local `RBD mirror deamon` status?